### PR TITLE
Fix recursive call of CreateError.to_string

### LINKED ISSUE #120: Using an actor name with parentheses causes an abort
I called a `Context::actor_of` with a name containing `(` and `)` (I used `&format!("{:?}"`) and determined that it was the `()` that caused it.

```
$ cargo run
   Compiling actix v0.1.0 (/Users/anacrolix/src/eratosthenes/rust/riker)
    Finished dev [unoptimized + debuginfo] target(s) in 2.68s
     Running `target/debug/actix`
2020-06-30 04:45:30+00:00 DEBG [riker::system] Starting actor system: System[riker]
2020-06-30 04:45:30+00:00 DEBG [riker::system] Actor system [ea287462-b746-48e2-a584-07db5bfdb1f1] [riker] started
Int(2)

thread 'pool-thread-#10' has overflowed its stack
fatal runtime error: stack overflow
Abort trap: 6
```

### PR DESCRIPTION
**Pull request by anacrolix**
Fixes #120.

### CODE REVIEW

File: src/actor.rs [hunk 1]
```diff
@@ -116,22 +117,7 @@ impl Error for CreateError {
 
 impl fmt::Display for CreateError {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        match *self {
-            CreateError::Panicked => f.write_str(&self.to_string()),
-            CreateError::System => f.write_str(&self.to_string()),
-            CreateError::InvalidName(ref name) => {
-                f.write_str(&format!("{} ({})", self.to_string(), name))
-            }
```
Comment (igalic) [inline src/actor.rs:124]: will this information still be retained?

**Comment by olexiyb** [comment]
Hey, please fix lint issue.

**Comment by igalic** [comment]
to answer my own question, after looking at the full definition of the enum, the answer should be "yes"

File: src/actor.rs (hunk 1 above)
Comment (anacrolix) [reply src/actor.rs:124]: Yes

**Comment by anacrolix** [comment]
> Hey, please fix lint issue.

@olexiyb What lint issue? I don't see any CI.

**Comment by igalic** [comment]
see https://github.com/riker-rs/riker#pre-commit

**Comment by leenozara** [comment]
Hi @anacrolix your PR is almost there, you just need to run `pre-commit run -a` (you'll need pre-commit installed first) then push the commit to your PR.

**Comment by olexiyb** [comment]
> > Hey, please fix lint issue.
> 
> @olexiyb What lint issue? I don't see any CI.

Looks like link between travis and github is broken again
https://travis-ci.org/github/riker-rs/riker/builds/703738615

**Comment by anacrolix** [comment]
Thanks, I can't get to this until next week.

**Comment by leenozara** [comment]
No worries, I'll get your branch and do it. Thanks for this fix!

**Comment by anacrolix** [comment]
Thanks!

**Pull request merged**